core/variant/variant_utility.cpp

```diff
@@ -866,6 +866,13 @@ String VariantUtilityFunctions::error_string(Error error) {
 	return String(error_names[error]);
 }
 
+String VariantUtilityFunctions::type_string(Variant::Type p_type) {
+#ifdef DEBUG_ENABLED
+	ERR_FAIL_INDEX_V_MSG((int)p_type, (int)Variant::VARIANT_MAX, "<invalid type>", "Invalid type argument to type_string(), use the TYPE_* constants.");
+#endif
```

Comment:
It's probably worth keeping this in non-debug builds, since this is mostly a developer utility anyway.